db scripts/create_new_user.sql

```diff
@@ -0,0 +1,31 @@
+CREATE OR REPLACE FUNCTION public.create_new_user(
```

Comment:
maybe upload the dll for the creation of the new tables as well?

Reply:
or rather update the existing DLL in db scripts/

Reply:
As far as I know there aren’t any dlls, maybe we should update all of them together

Reply:
definitely should sit in git

Reply:
since dll is also code i think it should be uploaded w/ the pr

Reply:
to allow commenting certain things (for example enforcing 'ON CASCADE' on foreign key delete) etc...